packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -520,18 +522,22 @@ class Frame extends React.Component {
    * Falls back to the string 'Search' if no label is provided
    */
   ariaLabel() {
-    const { ariaLabel, disabled, intl } = this.props;
+    const {
+      ariaLabel,
+      disabled,
+      placeholder,
+      intl,
+    } = this.props;
 
-    const defaultAriaLabel = intl.formatMessage({ id: 'Terra.form.select.ariaLabel' });
     const dimmed = intl.formatMessage({ id: 'Terra.form.select.dimmed' });
 
     // VO on iOS doesn't do a good job of announcing disabled stated. Here we append the phrase that
     // VO associates with disabled form controls.
     if ('ontouchstart' in window && disabled) {
-      return ariaLabel === undefined ? `${defaultAriaLabel} ${dimmed}` : `${ariaLabel} ${dimmed}`;
+      return ariaLabel === undefined ? `${placeholder} ${dimmed}` : `${ariaLabel} ${dimmed}`;
```

Comment:
what happens when placeholder is undefined OR not provided ?

Reply:
Default placeholder text (Select or Enter) will be render if placeholder is undefined or not provided.
<img width="1023" alt="Screenshot 2023-11-23 at 11 17 26 PM" src="https://github.com/cerner/terra-core/assets/127083404/2ee809fc-697f-4de6-8129-ae0221cc6bef">

Reply:
that is correct !! will that default value be available on ariaLabel function() ?? 
ariaLabel is reading values from props which has got default value has undefined

Reply:
why are not including placeholder text here when aria-label is present

Reply:
Update => https://github.com/cerner/terra-core/pull/3976/commits/522df473957cb63727028ea5c1164b160f8b08e4